src/main/java/com/iexec/core/task/update/TaskUpdateManager.java

```diff
@@ -348,7 +353,18 @@ void initialized2Running(Task task, ChainTask chainTask) {
         }
     }
 
-    void transitionFromRunningState(Task task, ChainTask chainTask) {
+    // region transitionFromRunningState
+
+    /**
+     * Handle transition when an off-chain task in the {@code RUNNING} state.
+     * <p>
+     * This method handles contribution timeouts, whether replicates list is empty and task transition to the next state
+     * depending on the task eligibility to {@code contributeAndFinalize} flow.
+     *
+     * @param chainTask On-chain task
+     * @param task      Off-chain task model of the on-chain task
+     */
+    void transitionFromRunningState(final ChainTask chainTask, final Task task) {
```

Comment:
Same question as my above comment (Why not just add the `final` and keep the original params order ?)